Add brpc and logstash files (#263)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/logstash.md

---
name: Logstash
category: Monitoring/Observability

> description: Logstash is a tool that helps collect, parse, and improve log and event data from different sources. It lets you manage data pipelines in real-time, providing insights and analysis for complex setups across multiple locations.

Author: ```suggestion
description: Logstash is a tool that helps to collect, parse, and improve log and event data from different sources. It also allows to manage data pipelines in real-time, providing insights and analysis for complex setups across multiple locations.
```

download_url: https://www.elastic.co/downloads/logstash
works_on_arm: true
supported_minimum_version:
    version_number: 7.9.0
    release_date: 2020/08/18

optional_info:
    homepage_url: https://www.elastic.co/logstash
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content: 

>         partner_content: https://amperecomputing.com/reference-architecture/deploying-an-elk-stack-on-google-tau-t2a

Author: it does not talk specifically about arm64 for logstash. please check if there is any other partner content.
